ddpui/api/webhook_api.py

```diff
@@ -123,7 +126,11 @@ def post_notification_v1(request):  # pylint: disable=unused-argument
     if send_failure_notifications:
         org = get_org_from_flow_run(flow_run)
         if org:
-            email_flowrun_logs_to_orgusers(org, flow_run["id"])
+            email_flowrun_logs_to_superadmins(org, flow_run["id"])
+            notify_platform_admins(org, flow_run["id"])
```

Comment:
For discord notifications to us (platform admins), can we add the org type whether its client or poc or demo or whatever the categories are. 

Also can we add the state whether its `failed` or `crashed`. Crashed should be a bigger concern for us.

Reply:
done